solutions/usaco-941.mdx

```diff
@@ -11,6 +11,37 @@ author: Jesse Choe, Ryan Chou, Chuyang Wang
 ## Video Solution
 <Youtube id="4YTijr0BbrA" />
 
+## Explanation
+
+First, it may help to think of an instance where we cannot form a proper evolutionary tree. This would be an instance such that no matter how we form the tree, it would be inevitable that some characteristic would evolve in two distinct places in the tree. It turns out that the minimal such bad example looks like this:
+
+$\texttt{population1: A } \\
+\texttt{population2: B } \\
+\texttt{population3: A B}$
+
+In other words, we have a population with just trait A, a population with just trait B, and a population with both. If we want to build a tree out of this input, we would need to split on either A or B at the root, but then the remaining two subtrees would both need to have an edge that adds the other characteristic. For example, if the root split into "A" and "not A" branches, then both branches would need to contain an edge that adds the B trait.
```

Comment:
oops i meant that you could (and probably should) still latex, just don't need texttt (so do `$A$` instead of `A` or `$\texttt{A}$`